src/main/java/com/xellitix/jenkins/plugintool/cli/CommandLineInterfaceModule.java

```diff
@@ -5,10 +5,7 @@
 import com.google.inject.TypeLiteral;
 import com.google.inject.multibindings.Multibinder;
 import com.google.inject.name.Names;
-import com.xellitix.jenkins.plugintool.cli.command.Command;
-import com.xellitix.jenkins.plugintool.cli.command.CommandHandler;
-import com.xellitix.jenkins.plugintool.cli.command.FetchInstalledPluginsCommand;
-import com.xellitix.jenkins.plugintool.cli.command.FetchInstalledPluginsCommandHandler;
+import com.xellitix.jenkins.plugintool.cli.command.*;
```

Comment:
Wildcard imports violate the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html). 
Please configure your IDE to not collapse multiple imports to a wildcard import and revert this change.